source/help/messaging/formatting-text.rst

```diff
@@ -125,23 +125,44 @@ Renders as:
 Code Block
 ----------
 
-Create a code block by indenting each line by four spaces, or by placing ``````` on the line above and below your code.
+Creating a fixed width code block is recommended for pasting multi-line blocks of code or other text output that is easier to read with fixed width font alignment. Examples include block text snippets, ASCII tables, log files, etc.
```

Comment:
Propose:

`...include block text snippets, ASCII tables, and log files. Create a code block by:

1. Placing three backtics (```) on the line directly above and directly below your code:
  [Example]
2. Indenting each line by four spaces:
[Example]

Reply:
agreed; and mostly adopted with some slight augmentation again